test/commands.test.js

```diff
@@ -1064,6 +1064,80 @@ test('[commands.operations.promptParameters] changesetParameters use previous va
   commands.operations.promptParameters(context);
 });
 
+test('[commands.operations.promptParameters] changesetParameters does not set UsePreviousValue when overrides set the value', function(assert) {
+  var oldParameters = { beep: 'bop' };
+  var newParameters = { beep: 'boop' };
```

Comment:
In the event that overrides are set, I think that the old and new values would reach this function looking identical. 

```
old === new === override
```

That's basically what caused the problem -- the old code would see that `old === new` and then `UsePreviousValue`, but it shouldn't do that when the override is set.

Reply:
yeah good catch. i've updated the tests to be reflect & be clear about this